packages/fx-core/src/plugins/resource/bot/resources/strings.ts

```diff
@@ -123,12 +123,10 @@ export const QuestionBotScenarioToBotCapability = new Map<BotScenario, BotCapabi
   [BotScenario.CommandAndResponseBot, BotCapabilities.COMMAND_AND_RESPONSE],
 ]);
 
-export const HostTypes = {
-  APP_SERVICE: "app-service",
-  AZURE_FUNCTIONS: "azure-functions",
-} as const;
-
-export type HostType = typeof HostTypes[keyof typeof HostTypes];
+export enum HostType {
+  APP_SERVICE = "app-service",
```

Comment:
It seems this is not the convention of Enum, maybe APP_SERVICE => AppService

Reply:
Since it is an enum, do we move to mapping.ts to keep all enum together?

Reply:
Done.